test/opexecuter.cpp

```diff
@@ -160,6 +160,9 @@ void OpExecuter::CompareNGraphAndTF() {
       case DT_INT64:
         AssertTensorEquals<int64>(tf_outputs_[i], ngraph_outputs_[i]);
         break;
+      case DT_BOOL:
+        AssertTensorEquals<bool>(tf_outputs_[i], ngraph_outputs_[i]);
+        break;
       default:
```

Comment:
This would get added once 'All' goes in

Reply:
Yes, I know but I needed it there to work on the op :)